tiny_dnn/core/kernels/fully_connected_op_avx.h

```diff
@@ -54,46 +54,46 @@ inline void avx_fully_connected_forward_kernel(const Tensor<float, S1> &in_data,
           _mm256_maskstore_ps(&*std::next(out, 8 * nblocks), imask, b);
         }
         for (serial_size_t c = 0; c < in_size; c++) {
-          auto in_val = _mm256_set1_ps(*std::next(in, c));
-          auto pW     = W.host_iter(0, c * out_size);
+          auto in_val = _mm256_set1_ps(in[c]);
+          auto pW     = W.host_pointer(0, c * out_size);
```

Comment:
Please get a pointer from W tensor outside of this loop.
We have no benefits from using host_pointer method inside loop like this.